app/src/main/res/layout/content_task.xml

```diff
@@ -41,7 +42,7 @@
                         android:textSize="24sp" />
 
                     <EditText
-                        android:id="@+id/task_title_textfield"
+                        android:id="@+id/filter_title_textfield"
```

Comment:
Let me guess, you first copied this card for the filter-card, and then used android-studio's ui to change the name of the textfield in the filter card? :D

Reply:
Fixed